sale_order_invoicing_queued/wizards/sale_advance_payment_inv.py

```diff
@@ -20,6 +20,10 @@ def enqueue_invoices(self):
         grouped_orders = defaultdict(lambda: order_obj.browse())
         for order in sale_orders:
             group_key = (order.partner_invoice_id.id, order.currency_id.id)
+            # If we have `sale_order_invoicing_grouping_criteria` module
+            # installed, we take that grouping criteria
+            if getattr(order, '_get_sale_invoicing_group_key'):
```

Comment:
```suggestion
            if hasattr(order, '_get_sale_invoicing_group_key'):
```

Reply:
Well, technically both works, as `getattr` return the method if exists and resolves as boolean `True`, but changed for being more meaningful.